storage/src/tests/storageserver/changedbucketownershiphandlertest.cpp

```diff
@@ -71,11 +67,17 @@ struct ChangedBucketOwnershipHandlerTest : Test {
     template <typename MsgType, typename... MsgParams>
     void expectDownAbortsMessage(bool expected, MsgParams&& ... params);
 
-    lib::ClusterState getDefaultTestClusterState() const {
+    std::shared_ptr<AbortBucketOperationsCommand> fetch_dispatched_abort_operations_command() {
+        _bottom->waitForMessages(2, 60); // abort cmd + set cluster state cmd
+        EXPECT_EQ(2, _bottom->getNumCommands()); // always _at least_ 2
```

Comment:
at least? GE?

Reply:
Exact equality is the intended semantics. I've removed this particular comment to avoid confusion.